tests/api-mongo.test.js

```diff
@@ -608,3 +609,24 @@ describe('MongoDB Words', () => {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  it('should return response status of 404 in /undefinedRoute', (done) => {
+    const route = '/undefinedRoute';
+    getAPIUrlRoute(route)
+      .end((_, res) => {
+        expect(res.status).to.equal(404);
+        done();
+      });
+  });
+
+  it('should contain Igbo API in / route', (done) => {
+    getAPIUrlRoute('/')
+      .end((_, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.be.an('object');
+        expect(res.text).to.contain('Igbo API');
```

Comment:
This test is really close! But I think we want a more definitive way of checking to see if we're getting the expected page. When you hit this route you should get the homepage. I think a better check is to see if we're getting back `html` from the backend.

Reply:
@pappyJ